src/content/advanced/gitops/apps/app_sets.md

```diff
@@ -0,0 +1,120 @@
+---
+linkTitle: Creating and using App Sets
+title: Creating and using App Sets
+description: Learn how to create and use App Sets for applications deployed with GitOps.
+weight: 50
+menu:
+  main:
+    identifier: advanced-gitops-apps-app-sets
+    parent: advanced-gitops-apps
+last_review_date: 2023-02-10
+user_questions:
+  - How can I create and use App Sets in GitOps?
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-honeybadger
+---
+
+# Creating and using App Sets
+
+It is often desireable to deploy a group of apps together, as a single deployment step. We call such groups "App Sets". There's nothing special about App Sets: they are not a separate API entity, rather just a configuration pattern enabled by `kustomize` and `flux`. The purpose of such approach can vary, but it's usually to meet the following benefits:
+
+- Apps within the set have strong relationship and dependency on versions and you want to be sure that the correct versions are used and deployed together.
+- Configuration of one app in a set depends on configuration of another app in the same set. By placing them together it is possible to simplify configuration of one app by providing values known to depend on another. As a result, it is easier to deploy an App Set than each of the apps individually. That way a specialized team might deliver a pre-configured App Set so that it is easy and understandable to deploy be a less proficient end user.
```

Comment:
```suggestion
- The configuration of apps in an App Set could be shared. By placing them together it is possible to simplify the configuration of one app by providing values known to depend on another. As a result, it is easier to deploy an App Set than each of the apps individually. That way a specialized team might deliver a pre-configured App Set so that it is easy and understandable to deploy be a less proficient end user.
```